packages/algorand-js/test/src/interpreter/opcode-list.ts

```diff
@@ -1055,6 +1056,64 @@ describe("Teal Opcodes", function () {
     });
   });
 
+  describe("Mulw", () => {
+    const stack = new Stack<StackElem>();
+
+    it("should return correct low and high value", () => {
+      stack.push(BigInt('4581298449'));
+      stack.push(BigInt('9162596898'));
+      const op = new Mulw();
+      op.execute(stack);
+
+      const low = stack.pop();
+      const high = stack.pop();
+      assert.equal(low, BigInt('5083102810200507970'));
+      assert.equal(high, BigInt('2'));
+    });
+
+    it("high bits should be 0", () => {
+      stack.push(BigInt('10'));
+      stack.push(BigInt('3'));
+      const op = new Mulw();
+      op.execute(stack);
+
+      const low = stack.pop();
+      const high = stack.pop();
+      assert.equal(low, BigInt('30'));
+      assert.equal(high, BigInt('0'));
+    });
+
+    it("low and high should be 0 on a*b if a or b is 0", () => {
```

Comment:
can you please add tests for big numbers too ?

Reply:
adding.